src/App.test.tsx

```diff
@@ -3,7 +3,7 @@ import {
   screen,
   waitForElementToBeRemoved,
 } from "@testing-library/react";
-import App from "./App";
+import App from "App";
```

Comment:
ini harusnya pake @/ aja biar seragam sama yang lain

```suggestion
import App from "@/App";
```

Reply:
okeee, tetep pake `@/` jadinya yaa meskipun hanya keluar 1 file